File: 백준/Bronze/2576. 홀수/홀수.cc
#include <stdio.h>

int main() {
    int n, min=100, count=0;
    
    for(int i=0;i<7;i++){
        scanf("%d", &n);
        if(n % 2 != 0){
            count += n;
            if(n <= min){min = n;}
        }
        n = 0;
    }
    if(count == 0){printf("-1");}
    else{printf("%d\n%d\n", count, min);}
    return 0;
}

File: 백준/Bronze/2576. 홀수/홀수.py
min, count = 100, 0
for i in range(0, 7):
    n = int(input())
    if n % 2 != 0:
        count += n
        if n <= min:
            min = n
    n = 0
if count == 0:
    print("-1")
else:
    print(count, min, sep='\n')